File: src/main/java/com/adjorno/billib/rest/db/Journal.java
package com.adjorno.billib.rest.db;

import javax.persistence.*;

@Entity
@Table(name = "JOURNAL")
public class Journal {
    @Id
    @Column(name = "_id")
    @GeneratedValue(strategy = GenerationType.AUTO)
    private Long mId;

    @Column(name = "NAME")
    private String mName;

    public Journal() {
    }

    public Journal(Long id, String name) {
        mId = id;
        mName = name;
    }

    public Long getId() {
        return mId;
    }

    public void setId(Long id) {
        mId = id;
    }

    public String getName() {
        return mName;
    }

    public void setName(String name) {
        mName = name;
    }
}

File: src/main/java/com/adjorno/billib/rest/db/Journal.kt
package com.adjorno.billib.rest.db

import jakarta.persistence.Column
import jakarta.persistence.Entity
import jakarta.persistence.GeneratedValue
import jakarta.persistence.GenerationType
import jakarta.persistence.Id
import jakarta.persistence.Table

@Entity
@Table(name = "JOURNAL")
data class Journal(
        @Id
        @Column(name = "_id")
        @GeneratedValue(strategy = GenerationType.IDENTITY)
        var id: Long? = null,

        @Column(name = "NAME")
        var name: String? = null
)
